packages/wallets/src/features/cfd/flows/Verification/Verification.tsx

```diff
@@ -139,15 +162,22 @@ const Verification: FC<TVerificationProps> = ({ selectedJurisdiction }) => {
                     return (
                         !formValues.drivingLicenceNumber ||
                         !formValues.drivingLicenseExpiryDate ||
+                        !!errors.drivingLicenseExpiryDate ||
                         !formValues.drivingLicenseCardFront ||
                         !formValues.drivingLicenseCardBack
                     );
                 } else if (formValues.selectedManualDocument === 'passport') {
-                    return !formValues.passportNumber || !formValues.passportExpiryDate || !formValues.passportCard;
+                    return (
+                        !formValues.passportNumber ||
+                        !formValues.passportExpiryDate ||
```

Comment:
I just update the validations to include `formValues.passportExpiryDate`, previously the `Next` button was enabled when the datepicker was not selecting the right format